src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -98,6 +109,39 @@ namespace OpenLoco::Vehicles::OrderManager
         }
     }
 
+    bool spaceLeftInGlobalOrderTableForOrder(const Order* order)
+    {
+        return numOrders() + kOrderSizes[enumValue(order->getType())] <= Limits::kMaxOrders;
+    }
+
+    bool spaceLeftInVehicleOrderTableForOrder(VehicleHead* head, const Order* order)
+    {
+        return head->sizeOfOrderTable + kOrderSizes[enumValue(order->getType())] < Limits::kMaxOrdersPerVehicle;
```

Comment:
This isn't actually the same as vanilla. This is saying only 63 bytes for orders allowed. Vanilla was 64 orders allowed (i.e. it used the ring view to count the number of orders).